src/app/components/home/partners/partners.component.ts

```diff
@@ -13,7 +13,24 @@ export class PartnersComponent implements OnInit {
   /**
    * Component constructor
    */
-  constructor() { }
+  items: Array<any> = []
+
+  constructor() {
+    this.items = [
+      { name: 'assets/images/partners/cmu.png' },
+      { name: 'assets/images/partners/facebook.png' },
+      { name: 'assets/images/partners/georgiatech.png' },
+      { name: 'assets/images/partners/google.png' },
+      { name: 'assets/images/partners/mapillary.png' },
+      { name: 'assets/images/partners/stanford.png' },
+      { name: 'assets/images/partners/virginiatech.png' },
+      { name: 'assets/images/partners/mapillary.png' },
+      { name: 'assets/images/thumb1.png' },
```

Comment:
thumb1 ?

Reply:
It was just a placeholder image to check the layout with multiple images. It can be removed.